ci-operator/config/openshift/must-gather-operator/openshift-must-gather-operator-master.yaml

```
@@ -11,27 +11,31 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: lint
+- as: validate
   commands: |
-    set -x
-    go mod download
-    make gocheck
+    make validate
   container:
     from: src
-- as: build
+- as: lint
   commands: |
-    set -x
-    go mod download
-    make gobuild
+    make lint
   container:
     from: src
-- as: unit
+- as: test
   commands: |
-    set -x
-    go mod download
     make test
   container:
     from: src
+- as: coverage
+  commands: |
```

```suggestion
  commands: |
    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
```
should we add this line?

---

Apparently so :)